### ISSUE TITLE
NVDA is announcing name twice when focus is on “Close” button in “Panel - Footer” flyout. 

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [microsoft/fluentui#12620]
User Experience:
Users who rely screen readers are getting impacted as they are listening the name twice when focus is on interactive elements.

Test Environment:
OS: Windows 10
Version: 1909(OS Build 18363.592)
Browser: Edge Canary (82.0.432.0)
URL: https://developer.microsoft.com/en-us/fluentui#/controls/web/panel
Screen Reader: NVDA 

Repro Steps:
Step 1: Turn On NVDA
Step 2: Open the above URL in Edge Canary browser.
Step 3: Now “Fabric UI - Panel" page will be displayed.
Step 4: Navigate to "Open Panel" control available in Panel - Footer section and activate it.
Step 5: Verify behaviour of the screen reader when focus is on “Close” button in “Improvement Actions” flyout. 

Actual:
NVDA is announcing name twice when focus is on “Close” button in “Panel - Footer” flyout.

Expected:
NVDA should announce name only once when focus is on “Close” button in “Panel - Footer” flyout.

MAS Reference:
MAS4.2.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

![image](https://user-images.githubusercontent.com/53210862/78858949-7b6b8000-7a4b-11ea-8ad0-946a5310b58e.png)

**Comment by joschect** [comment]
@SyedRehmanRazvi Could you confirm which name is being announced, is it the name of the close button or the title of the panel or something else? Does it happen with all panels, or only the footer panel? I tested it with Narrator and it seems to work appropriately so I suspect it's a problem with the NVDA reader. Also what are you referring to when you say `“Improvement Actions” flyout.` I can't find a reference to that.
